extensions/wikia/Recirculation/RecirculationHooks.class.php

```diff
@@ -81,7 +82,7 @@ public static function isCorrectPageType() {
 		$isInShowableNamespaces = $title->exists() && $title->inNamespaces( $showableNamespaces );
 
 		if ( $isInShowableNamespaces && !WikiaPageType::isActionPage() &&
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Replace this "if-then-else" statement by a single "return" statement. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1126)

[project-id]: wikia:app